### ISSUE TITLE
[Bug]: refreshTokenId not returned in the successful registration response

### ISSUE THREAD

**Issue by cezarneaga** [FusionAuth/fusionauth-issues#2928]
### What happened?

According to https://fusionauth.io/docs/apis/registrations#response-1 the refreshTokenId should be returned together with the refreshToken. in "^1.53.0" this isnt being returned. `refreshToken` is present

### Replication Steps

* Configure an application and allow it to generate refresh tokens (I did this both under the security tab for the login api and the oauth tab
* Run this curl script to add a new user and register the for the application
```
curl -H 'Authorization: APIKEY' -H 'Content-type: application/json' -X POST http://localhost:9011/api/user/registration -d '
{ 
 "registration": {
   "applicationId" : "85a03867-dccf-4882-adde-1a79aeec50df"
 },
  "user": {
    "password":"password",
    "email":"test2@example.com"
  }
}'
```

Expected: you get the refresh token id back, as documented.

Reality: you get the refresh token, not the id.

Internal: action/api/user/RegistrationAction.java#L240 looks like where we add the refresh token, but not the id, to the response.

### Version

1.53.2, 1.53.3

**Comment by mooreds** [comment]
@cezarneaga Thanks for the bug report! I'm able to replicate this. Looking to see if this is a doc bug or a functionality bug.

**Comment by cezarneaga** [comment]
this might be a separated issue, but in my head connected:

if the registration response has the AT&RT shouldn't it also have the `verificationId` for the email verification. i always assumed the reason for returning AT&RT is that you can automatically log user in on register.

**Comment by robotdan** [comment]
In your application configuration have you set `application.loginConfiguration.generateRefreshToken: true`?

**Comment by cezarneaga** [comment]
thanks, i am hoping that this can be appended to the response. it would save the users performing login. currently i am storing the refreshTokenId in a cookie instead of the refreshToken after user logs in, which i believe is a safer approach.

**Comment by cezarneaga** [comment]
> Interesting. The primary goal of returning the refreshTokenId is to provide the API caller the unique immutable Id that can be used to revoke the session (refresh token).

we use this to perform AT refresh & RT revoking.
we store the refreshTokenId in a cookie for 30 days. the accessToken is stored in a cookie but expiration is 60 min. 

we use refreshTokenId:
 to perform AT refresh, if the user doesn't explicitly logout.

**Comment by robotdan** [comment]
Assuming you didn't mean to close this until we fix it? 😎

**Comment by robotdan** [comment]
Internal:
- https://github.com/FusionAuth/fusionauth-app/pull/607

**Comment by cezarneaga** [comment]
Oh! Yes. I pressed enter and the default was the wrong button. Thanks for reopening

**Comment by robotdan** [comment]
A fix should go out in 1.54.0! Thanks for reporting!

**Comment by cezarneaga** [comment]
> A fix should go out in 1.54.0! Thanks for reporting!
@robotdan this didn't make it in the release cut 8 hrs ago?
